ftw/subsite/content/subsite.py

```diff
@@ -85,7 +85,32 @@
         name="FromEmail",
         widget=atapi.StringWidget(
             label=_(u'label_fromemail', default=u'Email Senderaddress'),
-            description=_(u'help_fromname', default=u'')))
+            description=_(u'help_fromname', default=u''))),
+
+    atapi.TextField(
+        name='webstatsAnalyticsSnippet',
+        schemata='subsite',
+        default_content_type='text/html',
+        allowable_content_types=('text/html',),
+        validators=('isTidyHtmlWithCleanup', ),
```

Comment:
I'm not sure if we should clean up here.
Usually the analytics include `<script>`-tags, which should usually be excluded.
Does script-tags work by default?